From e6a0cdf0b7045e7f9029cce287bcc539052ef79a Mon Sep 17 00:00:00 2001
From: Jonathan Backer <backer@chromium.org>
Date: Mon, 17 Jun 2019 16:46:37 +0000
Subject: [PATCH] Disallow partial swaps for linux based on gl_version_string

This was missed in https://chromium-review.googlesource.com/c/chromium/src/+/1601182

Bug: 973216,941670,957541,948640
Change-Id: I3d2002c38fe09d87c9a3db951dfc2ab112319caf
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1662528
Reviewed-by: Peng Huang <penghuang@chromium.org>
Commit-Queue: Peng Huang <penghuang@chromium.org>
Commit-Queue: Jonathan Backer <backer@chromium.org>
Auto-Submit: Jonathan Backer <backer@chromium.org>
Cr-Commit-Position: refs/heads/master@{#669701}
---
 gpu/config/gpu_driver_bug_list.json | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/gpu/config/gpu_driver_bug_list.json b/gpu/config/gpu_driver_bug_list.json
index 591a0642e8..e4a123ccd0 100644
--- gpu/config/gpu_driver_bug_list.json
+++ gpu/config/gpu_driver_bug_list.json
@@ -1865,10 +1865,6 @@
       },
       "gl_type": "gl",
       "gl_version_string": ".*Mesa.*",
-      "direct_rendering_version": {
-        "op" : "<",
-        "value": "2.3"
-      },
       "features": [
         "disable_post_sub_buffers_for_onscreen_surfaces"
       ]
